# SPDX-FileCopyrightText: Huawei Inc. 
# 
# SPDX-License-Identifier: Apache-2.0 

current START_ABILITY_TIMEOUT is too short, causing some
testcases in ActsBmsModuleUsageRecordTest, FeatureAbilityJsunit,
ActsBundleManagerTest to exceed timeout.
Increasing the timeout to 15 seconds.

Apply to test/xts/acts

Upstream-Status: Pending

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

diff --git a/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js b/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js
index b2a6034674..10e517f2c1 100644
--- a/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js
+++ b/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js
@@ -17,7 +17,7 @@ import wantconstant from '@ohos.ability.wantConstant'
 import { describe, beforeAll, beforeEach, afterEach, afterAll, it, expect } from 'deccjsunit/index'
 import commonEvent from '@ohos.commonevent'
 
-const START_ABILITY_TIMEOUT = 4000;
+const START_ABILITY_TIMEOUT = 15000;
 const TERMINATE_ABILITY_TIMEOUT = 1000;
 const TIMEOUT = 1000;
 var subscriberInfo_ACTS_StartAbility_0100 = {
diff --git a/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js b/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js
index d4068900a5..31ec534de7 100644
--- a/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js
+++ b/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js
@@ -24,7 +24,7 @@ const BUNDLE_NAME = 'com.example.third1';
 const NUM_TWO = 2;
 const NUM_TEN = 10;
 const INVALID_NUM = -1;
-const START_ABILITY_TIMEOUT = 3000;
+const START_ABILITY_TIMEOUT = 15000;
 const START_RECORD = 900;
 const TIMEOUT = 3000;
 
diff --git a/appexecfwk/bundle_standard/bundlemanager/actsbundlemanagertest/entry/src/main/js/test/ExampleJsunit.test.js b/appexecfwk/bundle_standard/bundlemanager/actsbundlemanagertest/entry/src/main/js/test/ExampleJsunit.test.js
index a4fcd2eb6b..e3a489c346 100755
--- a/appexecfwk/bundle_standard/bundlemanager/actsbundlemanagertest/entry/src/main/js/test/ExampleJsunit.test.js
+++ b/appexecfwk/bundle_standard/bundlemanager/actsbundlemanagertest/entry/src/main/js/test/ExampleJsunit.test.js
@@ -49,7 +49,7 @@ const SYSTEMDIR1 = "/data/accounts/account_0/applications/com.ohos.launcher/com.
 const CAMERA = 'com.permission.BMS_PERMISSION_CAMERA'
 const MUSIC = 'com.permission.BMS_PERMISSION_MUSIC'
 const WECHAT = 'com.permission.BMS_PERMISSION_WECHAT'
-const START_ABILITY_TIMEOUT = 3000;
+const START_ABILITY_TIMEOUT = 15000;
 var subscriberInfoEvent_0100 = {
     events: ['Third1_Publish_CommonEvent'],
 };
diff --git a/appexecfwk/bundle_standard/bundlemanager/actsbundlemanageruninstall/entry/src/main/js/test/ExampleJsunit.test.js b/appexecfwk/bundle_standard/bundlemanager/actsbundlemanageruninstall/entry/src/main/js/test/ExampleJsunit.test.js
index f4f1e6236d..0f5c37e4e4 100755
--- a/appexecfwk/bundle_standard/bundlemanager/actsbundlemanageruninstall/entry/src/main/js/test/ExampleJsunit.test.js
+++ b/appexecfwk/bundle_standard/bundlemanager/actsbundlemanageruninstall/entry/src/main/js/test/ExampleJsunit.test.js
@@ -36,7 +36,7 @@ const THIRD1 = "com.example.third1"
 const LAUNCHER = "com.ohos.launcher"
 const OBJECT = "object"
 const SUCCESS = "SUCCESS"
-const START_ABILITY_TIMEOUT = 3000;
+const START_ABILITY_TIMEOUT = 15000;
 var subscriberInfoEvent_0100 = {
     events: ['Third1_Publish_CommonEvent'],
 };
